src/factorization.jl

```diff
@@ -1,11 +1,20 @@
+function _ldiv!(x, A, b)
+    # work around https://github.com/JuliaLang/julia/issues/43507
+    if @which(ldiv!(x,A,b)) == which(ldiv!,Tuple{LU{Float64, Matrix{Float64}},Vector{Float64}}) 
```

Comment:
Does this resolve at compile time, or compile away?

Reply:
It does look like a runtime thing, so not ideal.

Reply:
we could just make it
```julia
function _ldiv!(x, A, b)
        copyto!(x, b)
        ldiv!(A, x)
end
```
and then it could be monkey-patched if need be.

Reply:
Or just

```julia
_ldiv!(x, A, b) = ldiv!(x, A, b)
function _ldiv!(x::Vector, A::Matrix, b::Vector)
        copyto!(x, b)
        ldiv!(A, x)
end
```

?

Reply:
`Factorization` instead of `Matrix`, but yeah